MediaSessionExt.cpp

```diff
@@ -1,16 +1,34 @@
+/*
```

Comment:
Is this the correct header?  If the file was taken from elsewhere where it has an RDK header, then this is OK but please append a credit to top level NOTICE:

Copyright 2020 RDK Management
Licensed under the Apache License, Version 2.0

If not, then please use the Metrological header as used elsewhere in this component.


There was a spurious code match as well which I have cleared off.

Reply:
Hi @mhughesacn 
this is a **work in progress** changes.
will look into your comment make required changes.